packages/web/components/Player/PlayerTile.tsx

```diff
@@ -58,7 +62,12 @@ export const PlayerTile: React.FC<Props> = ({ player }) => {
           key={player.id}
         >
           <VStack>
-            <PlayerAvatar player={player} page="list" />
+            <PlayerAvatar
+              player={player}
+              size="xl"
+              src={getPlayerImage(player)}
```

Comment:
You don't need to specify `src` or `name` since you are setting them from `player` in the component.